src/44bsd/tcp_socket.cpp

```diff
@@ -1100,19 +1100,48 @@ int utl_mbuf_buffer_create_and_copy(uint8_t socket,
 }
 
 
+/*
+ * The mbuf filled with patterns can be cached for other buffers.
+ * It will reduce 2K mbuf usage when lots of buffers are loaded.
+ * To avoid the 16bit refcnt overflow, const-mbuf is used.
+ * The const-mbuf could not be freed because it will stay in the driver's TX descriptor.
+ * In general, the number of cached mbufs is small and has no issue.
+ */
+static std::map<std::string,rte_mbuf_t*> mbufs_cached[MAX_SOCKETS_SUPPORTED];
```

Comment:
There is a leakage of mbufs in case you are using the cache. Where this map is clean and return the mbufs to the pools? 
more other in case that you move from enable -> disable there is no process that free the mbufs

Reply:
All the mbufs in mbufs_cached are const-mbuf. It should not be freed. It can be done safely at shutdown time.

Reply:
@jsmoon so if you are using interactive mode, you don't have a way to clean the cache?  will it stuck there forever? 
I think we need to give a way to clean the cache or disable this mode in interactive.